README.md

```diff
@@ -48,25 +48,45 @@ func main() {
         panic(err)
     }
     fmt.Printf("IP: %s released.", ip.IP)
+
+    // Now a IPv6 Super Prefix with Child Prefixes
+    prefix, err := ipamer.NewPrefix("2001:aabb::/48")
```

Comment:
Sample does not compile.

Should be:

```
	// Now a IPv6 Super Prefix with Child Prefixes
	prefix, err = ipam.NewPrefix("2001:aabb::/48")
	if err != nil {
		panic(err)
	}
	cp1, err := ipam.AcquireChildPrefix(prefix.Cidr, 64)
	if err != nil {
		panic(err)
	}
	fmt.Printf("got Prefix: %s.", cp1)
	cp2, err := ipam.AcquireChildPrefix(prefix.Cidr, 72)
	if err != nil {
		panic(err)
	}
	ip2, err := ipam.AcquireIP(cp2.Cidr)
	if err != nil {
		panic(err)
	}
	fmt.Printf("got IP: %s", ip2.IP)
```

And we should add "\n" at the end of each Printf-Statement